### ISSUE TITLE
Request for update Sunshine package

### ISSUE THREAD

**Issue by SoongVilda** [Jonatas-Goncalves/MaxxedSUSE#1]
Are you planning to update it? I really appreciate your repository because it covers most of the missing packages that I need, especially those that are great for games like:

[obs-studio](https://build.opensuse.org/package/show/home:MaxxedSUSE/obs-studio)
[sunshine](https://build.opensuse.org/package/show/home:MaxxedSUSE/sunshine) - that one need update, v0.22.0 version is out! https://github.com/LizardByte/Sunshine/releases/tag/v0.22.0

**Comment by Jonatas-Goncalves** [comment]
This beginning of the year has been full of changes and ended up taking me a lot of time, I intend to organize myself to update the entire repository here on Github with new apps and script updates by July, sorry about that.

I updated OBS and Sunshine, OBS had a problem with a VLC plugin that I fixed and updated to 30.0.2, and Sunhine had a conflict in libmfx that was also resolved, and is now updated to version 0.22.0.

Hope this helps.
